src/building/suggested.md

```diff
@@ -63,6 +63,33 @@ Task] instead:
 
 [Build Task]: https://code.visualstudio.com/docs/editor/tasks
 
+## Configuring `ctags`
+
+One of the challenges with rustc is that the RLS can't handle it, since it's a
+bootstrapping compiler. This makes code navigation difficult. One solution is to
+use `ctags`.
+
+`ctags` has a long history and several variants. Exuberant Ctags seems to be
+quite commonly distributed but it does not have out-of-box Rust support. Some
+distributions seem to use [Universal Ctags][utags], which is a maintained fork
+and does have built-in Rust support.
+
+The following script can be used to set up Exuberant Ctags:
+[https://github.com/nikomatsakis/rust-etags][etags].
+
+`ctags` integrates into emacs and vim quite easily. The following can then be
+used to build and generate tags:
+
+```console
+$ rust-ctags src/lib* && ./x.py build <something>
```

Comment:
If we end up keeping the ctags docs, this should be changed because `src/lib*` will not match anything in the compiler or standard library (only rustdoc).

Reply:
Based on looking at the script, I think that would require multiple invocations.